.golangci.yml

```diff
@@ -91,11 +92,6 @@ linters-settings:
         - github.com/elastic/beats/v7:
             reason: "There must be no Beats dependency"
 
-  gosimple:
```

Comment:
Why are you removing those linters?

Reply:
Because "config verify" tells me that the "go" option (and other options as well) does not longer exist for those linters.